dev/WindowsAppSDK_Insights/WindowsAppSDKInsightsLib/packages.config

```diff
@@ -1,5 +1,4 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.Telemetry.Inbox.Native" version="10.0.19041.1-191206-1406.vb-release.x86fre" targetFramework="native" />
```

Comment:
> <package id="Microsoft.Telemetry.Inbox.Native" version="10.0.19041.1-191206-1406.vb-release.x86fre" targetFramework="native" /> [](http://example.com/codeflow?start=1&length=128)

Do not remove. This is needed for the build scripts to work.

Reply:
Done.

Reply:
is this a private feed? we should not have private feed in github code.

Reply:
For Hui's question, redirecting to @ZenBird

Reply:
This has been addressed in a PR in main.

Reply:
link?

Reply:
Added you to the PR in main.